### ISSUE TITLE
Slow Text Generation on Desktop Compared to Mobile

### ISSUE THREAD

**Issue by Ahrimdon** [oobabooga/text-generation-webui#5202]
### Describe the bug

When using the `Mixtral-8x7B-Instruct-v0.1-GPTQ` model with the `ExllamaV2_HF` model loader on a desktop environment, text generation is noticeably slower compared to the mobile version of the WebUI. On the desktop, the generation speed is approximately a quarter of the speed observed on mobile (roughly 72% slower on desktop) on average, despite having a capable hardware setup and system resources barely being used. Trust remote code is enabled.

**Something to Note:** When running inference and generation on the desktop environment, the GPU usage ramps up to 100% for an extended period of time, which should be somewhat normal if the speeds matched it. However, when running on the Mobile WebUI, the GPU is barely used and only hits 100% when needed, reflecting a more normal expected outcome with the desired speeds to match.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

1.  Set up a Python environment with version 3.11.1.
2.  Upgrade pip in the virtual environment.
3.  Install necessary requirements using pip.
4.  Uninstall torch and torchvision with `pip uninstall torch torchvision`.
5.  Use pip to install specific versions of torch (`torch==2.1.2+cu121` and `torchvision==0.16.2+cu121`).
6.  Start the webfront using the parameters: `python server.py --listen --listen-host "0.0.0.0" --listen-port "7860" --api --api-port "5000" --verbose --trust-remote-code --character "Assistant" --auto-devices --chat-buttons --tensorcores --numa --model-dir "E:\AI" --loader "ExLlamav2_HF" --model "Mixtral-8x7B-Instruct-v0.1-GPTQ""`.
7.  Compare the text generation speeds using the `Mixtral-8x7B-Instruct-v0.1-GPTQ` model or other variations of Mixtral on a desktop (tested on both Ungoogled Chromium and Firefox browsers) and on a mobile device (tested on Safari browser on Apple iPhone).

### Screenshot

_Not applicable_

### Logs

Generation Parameters
```
{   'max_new_tokens': 50,
    'temperature': 0.7,
    'temperature_last': False,
    'dynamic_temperature': False,
    'dynatemp_low': 1,
    'dynatemp_high': 1,
    'dynatemp_exponent': 1,
    'top_p': 0.9,
    'min_p': 0,
    'top_k': 20,
    'repetition_penalty': 1.15,
    'presence_penalty': 0,
    'frequency_penalty': 0,
    'repetition_penalty_range': 1024,
    'typical_p': 1,
    'tfs': 1,
    'top_a': 0,
    'guidance_scale': 1,
    'penalty_alpha': 0,
    'mirostat_mode': 0,
    'mirostat_tau': 5,
    'mirostat_eta': 0.1,
    'do_sample': True,
    'encoder_repetition_penalty': 1,
    'no_repeat_ngram_size': 0,
    'min_length': 0,
    'num_beams': 1,
    'length_penalty': 1,
    'early_stopping': False,
    'use_cache': True,
    'eos_token_id': [2],
    'logits_processor': []
}
```

Desktop *(1 Tab of Ungoogled Chromium)*:
`Output generated in 750.13 seconds (0.07 tokens/s, 49 tokens, context 56, seed 907633688)`
Mobile *(Safari)*:
`Output generated in 28.30 seconds (1.73 tokens/s, 49 tokens, context 56, seed 1417775442)`
* *Both used the same parameters*

### System Info

-   **Operating System**: Windows 10
-   **GPU Brand and Model**: NVIDIA 3060 12GB
-   **CPU**: i7-11700K
-   **RAM**: 32GB
-   **Python Version**: 3.11.1
-   **Torch**: 2.1.2+cu121
-   **Torchvision**: 0.16.2+cu121
